scripts/complete_liquidity_provision.js

```diff
@@ -109,9 +109,9 @@ module.exports = async (callback) => {
     const bracketAddresses = await deployFleetOfSafes(masterSafe.address, argv.fleetSize, true)
     console.log("List of bracket traders in one line:", bracketAddresses.join())
 
-    // Sleeping for 1 seconds to make sure Infura nodes have processed all newly deployed contracts so that
+    // Sleeping for 3 seconds to make sure Infura nodes have processed all newly deployed contracts so that
     // they can be awaited.
-    await sleep(1000)
+    await sleep(3000)
```

Comment:
Does this work for sure? How did we deduce this number?

Reply:
Empirically,

No, there is no guarantee. A better fix would be to talk to only one node, but then the script does no longer become executable for everyone, as not everyone has access to a single node. Infura is just the industry standard.